src/auro-flight-main.js

```diff
@@ -36,7 +36,8 @@ class AuroFlightMain extends LitElement {
       reroutedDepartureStation: { type: String },
       reroutedArrivalStation: { type: String },
       duration:         { type: Number },
-      flights:          { type: Array }
+      flights:          { type: Array },
+      stops:            { type: Array }
```

Comment:
With the addition of this attribute, this needs to be documented in the JSdoc descriptions.

Reply:
updated

Reply:
How's this look?